test/shared/src/main/scala/zio/test/Assertion.scala

```diff
@@ -443,6 +443,12 @@ object Assertion {
     BoolAlgebra.failure(AssertionValue(self, actual))
   }
 
+  /**
+   * Makes a new assertion that search a substring.
```

Comment:
Typo. Also can we put it in alphabetical order so it is easy for people to find in the future?